src/loader.cpp

```diff
@@ -116,6 +116,15 @@ Mesh* Loader::load_stl()
         return NULL;
     }
 
+    qint64 file_size, file_size_old;
+    file_size = file.size();
+    do {
+        file_size_old = file_size;
+        usleep(100000);
```

Comment:
`usleep` is not available on Windows with MSVC

Reply:
Did I get the syntax wrong? My intention was to use https://doc.qt.io/qt-5/qthread.html#usleep.

Reply:
Oh interesting I wasn't aware of this function! Sorry about that

The approach that was merged uses the STL but i guess we should have looked into Qt